examples/cpp_control_interface/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ghcr.io/eclipse-ankaios/app-ankaios-dev:latest as dev
```

Comment:
Avoid using the `latest` tag but use a specific tag. Same for the other dev containers.

Reply:
Maybe we can reuse the same image if we remove the "Ankaios binaries download & install (the curl command)" at the docker build step to an entrypoint. The examples were already released with a suitable version, e.g. `v0.2.0`. If someone wants to run a example specific to a previous version then he can specify just the same version as the example is via parameter when running the image. Also for CI / CD it can be simplified with just one container image. The binaries built from main must only be copied into the container. No need to build a lot of images.

Reply:
But just as suggestion